tests/trips/test_trips.py

```diff
@@ -0,0 +1,98 @@
+from django.contrib.auth.models import User
+from django.urls import reverse
+
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase
+
+from api.modules.city.model import City
+from api.modules.trips.model import Trip
+
+
+class TestRemoveUserFromTripAPI(APITestCase):
+    """
+        Test for remove-user-from-trip API
+    """
+    api_url_name = "remove-user-from-trip"
+
+    @classmethod
+    def setUpTestData(cls):
+        """
+            run only once for all test methods
+            create user and related data objects
+        """
+        cls.current_user = User.objects.create_user(
+            "test_user1",
+            "user1@test.com",
+            "Django@123")
+        cls.current_user_token = Token.objects.create(user=cls.current_user)
+
+        cls.friend_user = User.objects.create_user(
+            "test_user2",
+            "user2@test.com",
+            "Django@123")
+
+        cls.city = City.objects.create(
+            city_name="test_city",
+            latitude=12.34,
+            longitude=12.34,)
+
+    def setUp(self):
+        """
+            run for each test method
+            create a new trip
+            sign in current-user
+        """
+        self.trip = Trip.objects.create(
+            trip_name="test_trip",
+            city=self.city,)
+        self.api_authentication(self.current_user_token)
```

Comment:
You can have a look at this documentation as well
http://www.django-rest-framework.org/api-guide/testing/#forcing-authentication